src/universal/modules/userDashboard/containers/UserColumns/UserColumnsContainer.js

```diff
@@ -66,11 +80,15 @@ export default createFragmentContainer(
   connect(mapStateToProps)(UserColumnsContainer),
   graphql`
     fragment UserColumnsContainer_viewer on User {
+      userId: id
```

Comment:
@mattkrick not sure that I need to include this instance of `userId` in the query, but it is used as a prop on the component.